packages/cart/src/models/line-item.ts

```diff
@@ -47,7 +50,7 @@ export default class LineItem {
   @Property({ columnType: "text", nullable: true })
   thumbnail?: string | null
 
-  @Property({ columnType: "text" })
+  @Property({ columnType: "integer" })
```

Comment:
Would be nice to support float as quantity.

Reply:
should we use numeric and serializer?